app/views/achievements/_form.html.haml

```diff
@@ -10,17 +10,17 @@
         = f.text_area :description, class: "form-control"
     .form-group
       = f.label :points, t(:achievement_points), class: ["col-sm-2", "control-label"]
-      .col-sm-10
-        = f.text_field :points, class: "form-control"
+      .col-sm-2
+        = f.text_field :points, class: "form-control", type: "number", min: 1, max: 3, size: 5
```

Comment:
Fun fact: I actually briefly considered a negative achievement before. I was going to have a "-1: Play Account Siphon" achievement. I tweaked it to the current one instead.